R/jfa-methods.R

```diff
@@ -1496,3 +1496,362 @@ plot.jfaFairness <- function(x, type = c("estimates", "posterior", "robustness",
   p <- .theme_jfa(p, legend.position = if (length(unprivileged) == 1 && type == "posterior") "inside" else "top", legend.position.inside = c(0.8, 0.8))
   return(p)
 }
+
+# Methods for class: jfaFairnessSelection #####################################################
+
+#' @rdname jfa-methods
+#' @method print jfaFairnessSelection
+#' @export
+print.jfaFairnessSelection <- function(x, ...) {
+  if (!(x[["measure"]] == "error")) {
+    cat("The selected fairness measure is", x[["name"]], "\n\n")
+    if (!(is.null(x[["q1"]][["name"]]))) {
+      cat("The answer to the first question is:\t", x[["q1"]][["name"]], paste0("(", x[["q1"]][["value"]], ")\n"))
```

Comment:
Maybe add `(q1)` to this string (and the others) so that it becomes: `The answer to the first question (q1) is:\t`